### PR TITLE
Feat/delivery-time

### PR DESCRIPTION
**Pull request by xaviranik**
- Delivery time order details for vendor

### CODE REVIEW

File: templates/orders/details.php [hunk 1]
```diff
@@ -209,6 +214,12 @@
                                 <span><?php esc_html_e( 'Order Date:', 'dokan-lite' ); ?></span>
                                 <?php echo esc_html( dokan_get_date_created( $order ) ); ?>
                             </li>
+                            <?php if( ! empty( $delivery_time_date ) ): ?>
```
Comment (nurul-umbhiya) [inline templates/orders/details.php:217]: also check if module exists and if exists module is active

File: templates/orders/details.php [hunk 2]
```diff
@@ -261,6 +262,8 @@
                 </div>
             </div>
 
+            <?php do_action( 'dokan_order_details_general_details_after', $order_id ); ?>
```
Comment (shohag121) [inline templates/orders/details.php:265]: Getting merge conflict here. please check.

**Review by shohag121 — CHANGES_REQUESTED** [review]

File: includes/functions.php [hunk 3]
```diff
@@ -3989,6 +3989,39 @@ function dokan_format_date( $date = '', $format = false ) {
...
+ * Get a formatted date, time from WordPress format
+ *
+ * @param string|timestamp $date the date string or timestamp
+ * @param string|bool $format date format string or false for default WordPress date
+ * @since DOKAN_PRO_SINCE
+ *
+ * @throws Exception
+ * @return string|false The date, translated if locale specifies it. False on invalid timestamp input.
+ */
+function dokan_format_date_time( $date = '', $format = false ) {
```
Comment (nurul-umbhiya) [inline includes/functions.php:4002]: change function name with dokan_format_datetime()

File: includes/Assets.php [hunk 4]
```diff
@@ -840,6 +846,7 @@ public function get_admin_localized_scripts() {
                 'i18n_remove_personal_data_notice'    => __( 'This action cannot be reversed. Are you sure you wish to erase personal data from the selected orders?', 'dokan-lite' ),
                 'decimal_point'                       => $decimal,
                 'mon_decimal_point'                   => wc_get_price_decimal_separator(),
+                'i18n_date_format'                    => get_option( 'date_format' ),
```
Comment (nurul-umbhiya) [inline includes/Assets.php:849]: use wc_date_format() instead of get_option( 'date_format' )

**Review by nurul-umbhiya — CHANGES_REQUESTED** [review]

**Review by shohag121 — APPROVED** [review]

**Pull request merged**